src/cascadia/TerminalApp/TerminalPage.h

```diff
@@ -94,15 +94,15 @@ namespace winrt::TerminalApp::implementation
         void _HookupKeyBindings(TerminalApp::AppKeyBindings bindings) noexcept;
         void _RegisterActionCallbacks();
 
-        void _UpdateTitle(std::shared_ptr<Tab> tab);
-        void _UpdateTabIcon(std::shared_ptr<Tab> tab);
+        void _UpdateTitle(const winrt::com_ptr<Tab>& tab);
+        void _UpdateTabIcon(const winrt::com_ptr<Tab>& tab);
```

Comment:
Yea I think the refcount inc/dec before was unnecessary. These probably could just be `const Tab&`

_for these two, not `_RegisterTerminalEvents`_

Reply:
I can do `const Tab&` for `_UpdateTitle`, but I can only do `Tab&` for `_UpdateTabIcon`, since it'll call `tab.UpdateIcon()` and actually change the tab object directly.